Wyvarn/calamus
# Update an existing blog

## Description

This allow a user who is authenticated & authorized & who has writer role privileges to update an existing blog.

Also includes linting fixes

Fixes #29

## How can we test this?

Run application as indicated in the README
+ Make a request to first create a blog. `/api/v1/writer/blog`
+ Now make an update to any of the fieds created by making a PATCH request to `/api/v1/writer/blog{ID}`, passing the ID as the id of the blog from the previous response
+ you should get back a response that updates the blog that was initially created. This should be confirmed by checking the MongoDB running instance collection.

## Checklist:
